components/Gradient2Button.tsx

```diff
@@ -0,0 +1,36 @@
+import { PropsWithChildren } from "react";
+import { twMerge } from "tailwind-merge";
+
+interface Gradient2ButtonProps {
+  onClick?: () => void;
+  className?: string;
+  outerDivClassName?: string;
+}
+
+const Gradient2Button = ({
```

Comment:
You could just use the `GradientButton`. It's actually the same thing as `Gradient2Button` from ousd.com but with the oeth.com gradient color.